core/src/main/scala/org/http4s/internal/package.scala

```diff
@@ -123,68 +126,52 @@ package object internal {
   }
 
   // Adapted from https://github.com/typelevel/cats-effect/issues/199#issuecomment-401273282
-  @deprecated(
-    "Replaced by cats.effect.Async.fromFuture. You will need a ContextShift[F].",
-    "0.21.4")
+  @deprecated("Replaced by cats.effect.Async.fromFuture", "0.21.4")
```

Comment:
I'm guessing this should just be removed entirely since this has to be a bincompat breaking release anyway.

Reply:
:+1: I like to err on the side of keeping deprecations for graceful migrations, but this is private, so we can be more aggressive here.